bin/gpodder

```diff
@@ -37,15 +37,6 @@ from optparse import OptionGroup, OptionParser
 
 logger = logging.getLogger(__name__)
 
-try:
-    import dbus
-    have_dbus = True
-except ImportError:
-    print("""
-    Warning: python-dbus not found. Disabling D-Bus support.
```

Comment:
This message is misleading because dbus is not optional and can't be disabled, which is why gpodder has fake dbus for Windows and Mac builds. I think it fails to import dbus after this point, and removing this warning message will make it harder to diagnose the problem.

Reply:
Should I move the try ... except and the message to `app.py`, where the next `import dbus` happens?

As the plan here is to eventually remove the use of dbus-python and port the needed things to GDBus from GLib, I would like to remove the dbus import from all files which do not need it.

Reply:
There is a non-zero chance a "new feature" that uses dbus could be inserted in the startup chain before dbus is replaced, but I don't see a case where `app.py` isn't always next.